odoo/addons/base/tests/test_pdf.py

```diff
@@ -83,7 +83,7 @@ def test_branded_file_writer(self):
 
         # Read the metadata of the newly created pdf.
         reader_buffer = io.BytesIO(branded_content)
-        pdf_reader = pdf.PdfFileReader(reader_buffer)
+        pdf_reader = pdf.PdfReader(reader_buffer)
```

Comment:
why not using OdooPdfReader here? does getAttachment have an impact here? Look like it should be ok.